Makefile

```diff
@@ -54,11 +54,11 @@ else ifeq ($(RELEASE), 1)
 CARGO_OSDK_ARGS += --release
 endif
 
-ifeq ($(INTEL_TDX), 1)
+ifeq ($(CVM_GUEST), 1)
 BOOT_METHOD = grub-qcow2
 BOOT_PROTOCOL = linux-efi-handover64
 CARGO_OSDK_ARGS += --scheme tdx
-CARGO_OSDK_ARGS += --features intel_tdx
+CARGO_OSDK_ARGS += --features cvm_guest
```

Comment:
Now that `cvm_guest` is a default feature, do still need to add this feature to `CARGO_OSDK_ARGS`?